src/figuresClasses.ts

```diff
@@ -1,19 +1,91 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: string = 'triangle';
 
-}
+  color: string;
+
+  a: number;
+
+  b: number;
+
+  c: number;
 
-export class Circle {
+  constructor(color: string, a: number, b: number, c: number) {
+    if (a <= 0 || b <= 0 || c <= 0 || a + b <= c || a + c <= b || b + c <= a) {
+      throw new Error('Invalid triangle');
+    }
 
+    this.a = a;
+    this.b = b;
+    this.c = c;
+    this.color = color;
+  }
+
+  getArea(): number {
+    const s = (this.a + this.b + this.c) / 2;
+    const area = Math.sqrt(s * (s - this.a) * (s - this.b) * (s - this.c));
+    const calc = Math.floor(area * 100) / 100;
```

Comment:
1. Let's give more descriptive names, than just `calc`
2. As you use this logic `Math.floor(area * 100) / 100` in multiple places, why don't you put it into separate function ?

Reply:
Sure, I have fixed these issues!